Tfe updates (#2215)
hashicorp/terraform-website: content/enterprise/before-installing/index.mdx

Terraform Enterprise requires **at least one** of the following Docker Engine configurations, in order of preference:

1. 17.06.2-ce, 18.09.x, 19.03.x, or 20.10.x with `runc` v1.0.0-rc93 or greater.
1. 20.10.x with `libseccomp` 2.4.4 or greater.
1. 17.06.2-ce, 18.09.x, 19.03.x, or 20.10.x using a [modified `libseccomp` profile](https://www.terraform.io/enterprise/before-installing#option-2-docker-engine-with-a-compatible-libseccomp-version).

> If you are installing on RHEL7, you can use Docker Engine 1.13.1 from the Extras Repo, with a [modified `libseccomp` profile](https://www.terraform.io/enterprise/before-installing#option-2-docker-engine-with-a-compatible-libseccomp-version).

Reviewer: ```suggestion
If you are installing on RHEL7, you can use Docker Engine 1.13.1 from the Extra Packages for Enterprise Linux (EPEL) repository, with a [modified `libseccomp` profile](https://www.terraform.io/enterprise/before-installing#option-2-docker-engine-with-a-compatible-libseccomp-version).
```

1. Docker Engine 17.06.2-ce, 18.09.x, 19.03.x, or 20.10.x with `runc` v1.0.0-rc93 or greater.
1. Docker Engine 20.10.x with `libseccomp` 2.4.4 or greater.

> 1. 17.06.2-ce, 18.09.x, 19.03.x, or 20.10.x using a [modified `libseccomp` profile](https://www.terraform.io/enterprise/before-installing#option-2-docker-engine-with-a-compatible-libseccomp-version).

Reviewer: I see you removed "Docker Engine" which I agree with since we specify that they are all docker engines above. Can we remove that from the first two bullets, too? also! Is this a repeat of the first option?

Author: I'll make that update. And no, it's different from 1. 1 is any of those version combined with the specified version of runc. The other one is if you don't have that updated runc installed, you instead need the modified libseccomp profile. My brain hasn't come up with a better way to display this data, but I am open to suggestions.

Reviewer: okay I'll leave as is for now :)

> If you are installing on RHEL7, you also have the option of Docker Engine 1.13.1 from the Extras Repo, with a [modified `libseccomp` profile](https://www.terraform.io/enterprise/before-installing#option-2-docker-engine-with-a-compatible-libseccomp-version).

Reviewer: I'm also wondering where this falls in order of preference. The ones above are listed in order of most to least recommended. Should folks on RHEL7 use this one, or would we recommend one of the above?

```suggestion
If you are installing on RHEL7, you can also use Docker Engine 1.13.1 from the Extras Repo, with a [modified `libseccomp` profile](https://www.terraform.io/enterprise/before-installing#option-2-docker-engine-with-a-compatible-libseccomp-version).
```

Author: Yeah, that's the least preferred option as it's old and not kept as up to date as everything else.
